notifications.bs

```diff
@@ -211,6 +217,24 @@ string <var>title</var>, {{NotificationOptions}} <a for=/>dictionary</a> <var>op
  <li><p>Set <var>notification</var>'s <a for=notification>body</a> to
  <var>options</var>["{{NotificationOptions/body}}"].
 
+ <li>
+  <p>If <var>options</var>["{{NotificationOptions/url}}"] <a for=map>exists</a>:
+
+  <ol>
+   <li><p>Let <var>url</var> be the result of <a lt="URL parser">parsing</a>
+   <var>options</var>["{{NotificationOptions/url}}"] using <var>baseURL</var>.
+
+   <li>
+    <p>If <var>url</var> is not failure:
+
+    <ol>
+     <li><p>If <var>url</var>'s <a for=url>origin</a> is not <a>same origin</a> with
```

Comment:
Let's remove this check and the one below as JavaScript can already do whatever.